Automattic/jetpack
# Search package: duplicate dashboard dependencies to package (1/3)

Preceding PRs: #21814 #21584 

#### Changes proposed in this Pull Request:

Copied dependencies from Jetpack the plugin with zero change as a base. tests and state are skipped as we'll add new.

#### Jetpack product discussion
n/a

#### Does this pull request change what data or activity we track or use?
no

#### Testing instructions:
- Ensure there are no changes to any files against their originals. Folders on the left are the original locations, which is in `projects/jetpack/_inc/client`
    - `components/buttons` =>  `components/buttons`
    - `components/card` =>  `components/card`
    - `components/form/form-toggle` =>  `components/form-toggle`
    - `components/global-notices` =>  `components/global-notices`
    - `components/notice` =>  `components/notice`
